frontend/src/components/House/styles.js

```diff
@@ -0,0 +1,21 @@
+import {StyleSheet} from 'aphrodite';
+
+const styles = StyleSheet.create({
+  list: {
+    display: "flex",
+    flexDirection: "column"
+  },
+  wrapper: {
+    display: "flex",
+    flexDirection: "row",
```

Comment:
Its row by default.

Reply:
I concur, we don't need `flexDirection: "row", ` flexbox is by default in row mode, so we can remove the line. If you want. It doesn't really matter!